plugins/panel/plugin.js

```diff
@@ -297,8 +297,8 @@
 			markItem: function( index ) {
 				if ( index == -1 )
 					return;
-				var links = this.element.getElementsByTag( 'a' );
-				var item = links.getItem( this._.focusIndex = index );
+				var focusables = this._.getItems();
+				var item = focusables.getItem( this._.focusIndex = index );
```

Comment:
You could join these two subsequent `var` statements, as this is our code style since couple of years.